src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -59,15 +60,19 @@ public static <T> Logger getLogger(Class<T> clazz) {
         return getLogger(clazz.getSimpleName());
     }
 
-    /**
-     * Adds the {@code consoleHandler} to the {@code logger}. <br>
-     * Creates the {@code consoleHandler} if it is null.
-     */
-    private static void addConsoleHandler(Logger logger) {
-        if (consoleHandler == null) {
-            consoleHandler = createConsoleHandler();
+    private static Logger getBaseLogger() {
+        Logger baseLogger = Logger.getLogger(BASE_LOGGER_NAME);
+        baseLogger.setUseParentHandlers(false);
+        removeHandlers(baseLogger);
+        baseLogger.addHandler(createConsoleHandler(Level.ALL));
```

Comment:
Why use `Level.ALL` here?

Reply:
This is to ensure that all the handlers will log messages regardless of their level. So we dont need to track these loggers as a global variable in the class.

Since the logger is the one generating the messages, we dont want the level of the handlers to be higher than the log level of the loggers as this will cause the messages of the loggers to be ignored by the handler. (e.g. logger set to FINEST but handler is at INFO)

Since both handlers will always all messages by generated by the logger, there is no need to purposefully set the log level of each handler. Furthermore, setting these handlers to ALL won't impact performance since they are just comparing the log levels of the messages.